admin/c2cgeoportal_admin/views/notfound.py

```diff
@@ -0,0 +1,7 @@
+from pyramid.view import notfound_view_config
+
+
+@notfound_view_config(renderer='../templates/404.jinja2')
+def notfound_view(request):
```

Comment:
Do we relay needs a specific view for this error?

Reply:
I dunno. What do others think ? @arnaud-morvan ?

Reply:
This comes from the cookie cutter template, this give the 404 a ninja template, using the layout, I think it's a good thing for gmf_admin

Reply:
ok :-)